vitess.io/go-import/vitess.html

```diff
@@ -1,4 +1,6 @@
 <html><head>
 <meta name="go-import" content="vitess.io/vitess git https://github.com/vitessio/vitess">
 <meta name="go-source" content="vitess.io/vitess     https://github.com/vitessio/vitess https://github.com/vitessio/vitess/tree/master{/dir} https://github.com/vitessio/vitess/blob/master{/dir}/{file}#L{line}">
+<meta name="go-import" content="vitess.io/messages git https://github.com/vitessio/messages">
```

Comment:
The process for adding a new redirect is more like:

* copy `go-import/vitess.html` to `go-import/messages.html` and update the URLs
* edit `_redirects` and copy the line related to `/vitess/*` and change it to refer to `/messages/*`